tests/contrib/awards/test_awards_datastreams.py

```diff
@@ -124,6 +124,7 @@ def test_awards_service_writer_create(
     award_dict = award_rec.entry.to_dict()
 
     award_full_data["funder"]["name"] = example_funder_ec["name"]
+    award_full_data.pop("pid")
```

Comment:
shelvable to be able to merge this but I think is important: shouldn't the data be accepted as `id` instead of pid? otherwise we have the payload mistmatch between create/update.